doc/k8s/sqlflow-server.yaml

```diff
@@ -12,7 +12,7 @@ spec:
   template:
     metadata:
       labels:
-        app: sqlflow-server
+        app: sqlflow-server:latest
```

Comment:
I believe the default Docker image tag is `latest`, can you mention why using a specified Docker tag?